autocertkit/network_tests.py

```diff
@@ -1002,6 +1002,85 @@ def test_ping(self, session):
         return self._run_test(session)
 
 
+class MulticastTest(IperfTest):
+    """ Subclass that adds multicast specific operations"""
+    MULTICAST_IP = '226.94.1.1'
+    iperf_report_path = '/tmp/iperf.log'
+
+    def run_mulitcast_server(self):
+        """Start the iPerf server listening on a VM"""
+        log.debug('Starting multicast server')
+        self.server_ip = self.get_server_ip('eth0')
```

Comment:
@bpwook Dom0 certainly supports multicast, this test case just for ensuring that multicast traffic from a VM to another VM on another host works.